index.html

```diff
@@ -9367,6 +9406,7 @@ <h5>Presentational Role Inheritance</h5>
 				<p>The toolbar is often a subset of functions found in a <rref>menubar</rref>, designed to reduce user effort in using these functions. Authors MUST supply a label on each toolbar when the application contains more than one toolbar.</p>
 				<p>Authors MAY manage focus of descendants for all instances of this <a>role</a>, as described in <a href="#managingfocus">Managing Focus</a>.</p>
 				<p>Elements with the role <code>toolbar</code> have an implicit <pref>aria-orientation</pref> value of <code>horizontal</code>.</p>
+				<p>Authors MAY set <pref>aria-setsize</pref> and <a>aria-posinset</a> on focusable interactive <a>accessibility children</a> of the role <code>toolbar</code> that don't have a <a href="#scope">required accessibility parent role</a>. Authors MUST NOT set <pref>aria-setsize</pref> and <a>aria-posinset</a> values scoped to the <code>toolbar</code> on <a href="#mustContain">allowed accessibility child roles</a> of any <rref>composite</rref> <rref>widget</rref> that already provides indexing.</p>
```

Comment:
porting over the previous suggestion:

```suggestion
                <p>Authors MAY set <pref>aria-setsize</pref> and <pref>aria-posinset</pref> on focusable interactive <a>accessibility children</a> of the role <code>toolbar</code> or <a>accessibility descendants</a> with only elements with role <rref>group</rref> intervening. If the <code>toolbar</code> contains controls that already support <pref>aria-posinset</pref> and <pref>aria-setsize</pref> (such as <rref>option</rref> in a <rref>listbox</rref> or <rref>menuitem</rref> in a <rref>menu</rref>), authors SHOULD treat any provided <pref>aria-posinset</pref> and <pref>aria-setsize</pref> values on those descendants as fully independent from the toolbar scope.</p>
                <p>Unlike other roles that support <pref>aria-setsize</pref> and <pref>aria-posinset</pref> such as <rref>listitem</rref> or <rref>menuitem</rref>, browsers are not required to calculate implicit values scoped to a <code>toolbar</code>.</p>
```